flink-python/tox.ini

```diff
@@ -21,7 +21,7 @@
 # in multiple virtualenvs. This configuration file will run the
 # test suite on all supported python versions.
 # new environments will be excluded by default unless explicitly added to envlist.
-envlist = {py37, py38, py39, py310}-cython
```

Comment:
The current fix does not work for python 3.7 because there is no 1.19 package released for this python version.
Since dependency on `apache-flink` python package is set as `apache-flink>=1.17.0` - only the latest available Flink version is used during testing.